client/my-sites/sidebar-unified/item.jsx

```diff
@@ -51,16 +52,15 @@ export const MySitesSidebarUnifiedItem = ( {
 			forceInternalLink
 			className={ isSubItem ? 'sidebar__menu-item--child' : 'sidebar__menu-item-parent' }
 		>
-			{ children }
+			<MySitesSidebarUnifiedStatsSparkline slug={ slug } />
```

Comment:
By passing the memoized component `MySitesSidebarUnifiedStatsSparkline` we can avoid causing `SidebarItem` to re-render due to a change in `children`.